cb_defense_syslog.py

```diff
@@ -336,31 +336,20 @@ def parse_notifications(server, notifications_response, audit_response):
     notifications_log = None
     audit_log = None
 
-    threat_hunter = n.psc_threathunter_check(notifications_response)
-
     if config.get('general', 'output_format') not in accepted_formats:
         return None
 
     if notifications_response != None and audit_response != None:
```

Comment:
This is nit picking a little bit. Could this be separated slightly without changing the logic?

```
notifications_log = None
audit_log = None
if notifications_response is not None:
    parse()

if audit_response is not None:
   parse()
```

Reply:
Changes made. Updates pushed